nix-darwin/default.nix

```
@@ -9,7 +9,10 @@ let
   extendedLib = import ../modules/lib/stdlib-extended.nix pkgs.lib;
 
   hmModule = types.submoduleWith {
-    specialArgs = { lib = extendedLib; };
+    specialArgs = { 
+      lib = extendedLib; 
+      nixosConfig = config;
```

```suggestion
      darwinConfig = config;
```

Feels more appropriate.